var/spack/repos/builtin/packages/sim-model/package.py

```diff
@@ -70,11 +70,14 @@ def __build_mods_coreneuron(self, mods_location, link_flag, include_flag):
         spec = self.spec
         assert os.path.isdir(mods_location), mods_location
         include_flag += ' -I%s' % (spec['coreneuron'].prefix.include)
-        which('nrnivmodl-core')(
-            '-i', include_flag, '-l', link_flag, '-n', self.mech_name,
-            '-v', str(spec.version), '-c', mods_location)
+        if spec.satisfies('^coreneuron@0:0.16'):
+            which('nrnivmodl-core')(
+                '-i', include_flag, '-l', link_flag, '-n', self.mech_name,
+                '-v', str(spec.version), '-c', mods_location)
+        else:
+            which('nrnivmodl-core')('-i', include_flag, '-l', link_flag, mods_location)
```

Comment:
Newer coreneurons dont accept versioning the library?

Reply:
yeah, we discussed that offline.